src/main/java/de/tum/in/www1/artemis/web/rest/AeolusTemplateResource.java

```diff
@@ -25,7 +25,6 @@
  */
 @Profile("aeolus | localci")
 @RestController
-@RequestMapping("api/aeolus")
```

Comment:
Is it possible to add another/a newly created annotation to keep the "aeolus" here?